src/redux/books/books.js

```diff
@@ -0,0 +1,50 @@
+import { createReducer, createAction } from '@reduxjs/toolkit';
+
+export const ADD_BOOK = createAction('books/add');
+export const REMOVE_BOOK = createAction('book/remove');
+
+const initialState = {
+  books: [
+    {
+      id: '1',
+      title: 'The Hunger Games',
+      author: 'Suzanne Collins',
+      category: 'Action',
+    },
+    {
+      id: '2',
+      title: 'Dune',
+      author: 'Frank Herbert',
+      category: 'Science Fiction',
+    },
+    {
+      id: '3',
+      title: 'Capital in the Twenty-First Century',
+      author: 'Suzanne Collins',
+      category: 'Economy',
+    },
+  ],
+};
+
+const booksReducer = createReducer(initialState, (builder) => {
+  builder.addCase(ADD_BOOK, (state, action) => {
+    const updatedState = {
+      ...state,
+      books: [
+        ...state.books,
+        { ...action.payload, id: `${state.books.length + 1}` },
+      ],
+    };
+    return updatedState;
+  });
```

Comment:
- [ ] Wonderful job for putting your project this far. However, there's few things you need to do for this part to meet the project requirement. The user should be able to add books which is possible when I tried it, but the added books disappeared when the page reload. Kindly look into that and fix. Thanks.

[add-book.webm](https://user-images.githubusercontent.com/26597432/215263129-c897da69-2511-41df-830b-2222c16ec98d.webm)

Reply:
Fixed.